src/features/notifications/NotificationListItem.tsx

```diff
@@ -0,0 +1,91 @@
+import React, { useMemo } from 'react'
+import Text from '../../components/Text'
+import Box from '../../components/Box'
+import TouchableOpacityBox, {
+  TouchableOpacityBoxProps,
+} from '../../components/TouchableOpacityBox'
+import { Notification } from '../../generated/graphql'
+import parseMarkup from '../../utils/parseMarkup'
+
+export type NotificationListItemProps = {
+  notification: Notification
+  viewed?: boolean
+  hasDivider?: boolean
+} & TouchableOpacityBoxProps
+
+const NotificationListItem = ({
+  notification,
+  viewed,
+  hasDivider,
+  ...rest
+}: NotificationListItemProps) => {
+  const title = useMemo(() => {
+    return notification.title
+  }, [notification.title])
+
+  const subtitle = useMemo(() => {
+    return parseMarkup(notification.body)
+  }, [notification.body])
+
+  const time = useMemo(() => {
+    const date = new Date(notification.time)
+    // Get hours with leading zero
+    const hours = date.getHours().toString().padStart(2, '0')
```

Comment:
This is interesting. Are we considering time zone?

Reply:
it should be UTC, not sure what we want to show to the user though

Reply:
I ended up going with local time and using AM/PM format